contracts/src/Gateway.sol

```diff
@@ -613,4 +620,33 @@ contract Gateway is IGateway, IInitializable, IUpgradable {
         assets.assetHubCreateAssetFee = config.assetHubCreateAssetFee;
         assets.assetHubReserveTransferFee = config.assetHubReserveTransferFee;
     }
+
+    // Calculate cost for transact
+    function _calculateTransactCost() internal pure returns (Costs memory costs) {
+        return Costs({native: 0, foreign: 0});
+    }
+
+    /// @inheritdoc IGateway
+    function sendCall(
+        ParaID destinationChain,
+        OriginKind originKind,
+        uint128 destinationFee,
+        Weight calldata weightAtMost,
+        bytes calldata call
+    ) external payable {
+        if (call.length == 0 || destinationFee == 0 || weightAtMost.refTime == 0 || weightAtMost.proofSize == 0) {
+            revert InvalidTransact();
+        }
+        bytes memory payload =
+            SubstrateTypes.Transact(msg.sender, originKind.encode(), destinationFee, weightAtMost, call);
+        Costs memory costs = _calculateTransactCost();
+        Ticket memory ticket = Ticket({dest: destinationChain, costs: costs, payload: payload});
+        _submitOutbound(ticket);
+    }
+
+    /// @inheritdoc IGateway
+    function quoteSendCallFee(uint128 destinationFee) external view returns (uint256) {
```

Comment:
`destinationFee` can be removed from here, as the quote should only return delivery fees.

Reply:
https://github.com/Snowfork/snowbridge/pull/1141/commits/bfdff266ac2acac0a39be8f3f82897a427b1dd72